capabilities/neuvector/zarf.yaml

```diff
@@ -20,26 +20,22 @@ components:
         namespace: neuvector
         gitPath: charts/core
         valuesFiles:
-          - values/neuvector-minimal-values.yaml
+          - values/values.yaml
       - name: monitor
         url: https://neuvector.github.io/neuvector-helm/
         version: 2.6.4
         namespace: neuvector
         gitPath: charts/monitor
         valuesFiles:
           - values/neuvector-monitor-values.yaml
+      - name: uds-neuvector-config
```

Comment:
not sure how I feel about combining the app portion with config things in one component. Realize it saves an extra import line, but don't know that this is necessarily the right move.

Reply:
I dont have a problem changing it back in each place, but at least in the current state it didnt seem like we are gaining anything from the config charts being separate except more import statements

Reply:
In general I like combining the config chart + app into a single component for simplicity of imports, but happy to hear compelling reasons on why that might not be a good option.

Reply:
yeah it might be fine, idk--worth just deciding one way and moving on. For Istio for now, we have to keep things separate because of an outstanding helm bug. But that's not a reason to force everything else that way. Originally, components were designed to logically group apps/behavior--so you could argue that one component is reasonable in that regard. It certainly would help the imports here.

Reply:
per several discussions, leaving the config-charts as part of the main components